oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -286,23 +280,31 @@ public void afterBulk(long executionId, BulkRequest bulkRequest, BulkResponse bu
         }
 
         @Override
-        public void afterBulk(long executionId, BulkRequest bulkRequest, Throwable throwable) {
-            LOG.error("ElasticIndex Update Bulk Failure : Bulk with id {} threw an error", executionId, throwable);
-            suppressedExceptions.add(throwable);
+        public void afterBulk(long executionId, BulkRequest request, List<String> contexts, Throwable failure) {
```

Comment:
If the methods in this listener fail with an exception, is the exception logged? This depends on the logic of the Elastic client, but it's important to know if Elastic will print the exception or if it suppresses it.

Reply:
when the bulk processor  gets closed if there are suppressed exceptions, they are grouped in a IOException that gets thrown. We have some tests around this (see `ElasticPropertyIndexTest#indexFailuresWithFailOnErrorOn`).